### PR TITLE
Fix AlphaFold2 workflow for multimer predictions

### PR DESCRIPTION
**Pull request by athbaltzis**
## PR checklist

- [X] This comment contains a description of changes (with reason).
- [X] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: workflows/alphafold2.nf [hunk 1]
```diff
@@ -72,17 +72,29 @@ workflow ALPHAFOLD2 {
...
+        INPUT_CHECK (
+            ch_input
+        )
+        .reads
+        .map {
+            meta, fasta ->
+            [ meta, fasta.splitFasta(file:true) ]
+        }
+        .transpose()
     .set { ch_fasta }
```
Comment (JoseEspinosa) [inline workflows/alphafold2.nf:85]:
```suggestion
       .set { ch_fasta }
```

File: workflows/alphafold2.nf [hunk 2]
```diff
@@ -72,17 +72,29 @@ workflow ALPHAFOLD2 {
...
+    } else {
+        INPUT_CHECK (
+            ch_input
+        )
+        .reads
+        .map {
+            meta, fasta ->
+            [ meta, fasta ]
+        }
+        .transpose()
```
Comment (JoseEspinosa) [inline workflows/alphafold2.nf:95]: Do we need this here? Is not returning just the same structure

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Very nice the `useGPU` parameter
Just make some small suggestions!

File: workflows/alphafold2.nf (hunk 2 above)
Comment (athbaltzis) [reply workflows/alphafold2.nf]: Yes, you are right! Fixed in the next commits

**Review by JoseEspinosa — APPROVED** [review]
🚀

**Pull request merged**